nupic/research/frameworks/pytorch/imagenet/network_utils.py

```diff
@@ -92,12 +92,19 @@ def create_model(model_class, model_args, init_batch_norm, device,
     model = model_class(**model_args)
     if init_batch_norm:
         init_resnet50_batch_norm(model)
-    model.to(device)
 
-    # Load model parameters from checkpoint
+    # backwards compatibility
```

Comment:
This comment is confusing. All you doing is checking if the optional `device` argument is `None`. It is confusing to mark that as 'backward compatibility'

Reply:
It is sending the model to device here, if device is not None. That is to maintain backward compatibility in case you only call `create_model` function directly and you expect it to send the model to device. In the new version, the model is only sent to device in the `transform_model` function. Do you propose a different approach?

Reply:
No, just remove the comment. I was trying to understand "backwards compatibility" comment from the code without looking at the diffs and it was not clear to me